### ISSUE TITLE
Ranger crashes when highlighting desktop.ini file on a Windows partition

### ISSUE THREAD

**Issue by Whitewing135** [ranger/ranger#1316]
#### Runtime Environment

- Operating system and version: Manjaro 17.1.12
- Terminal emulator and version: Konsole 18.08.1 (using qt5ct plugin)
- Python version: 3.7.0
- Ranger version/commit: ranger-master 1.9.1
- Locale: en_US.UTF-8

#### Steps to reproduce
1. Make fstab entry for the C windows partition.
2. Open ranger in ethier konsole or xterm
3. Select C:\User\Katal\Desktop\desktop.ini

#### Traceback

```
ranger version: ranger-master 1.9.1
Python version: 3.7.0 (default, Jul 15 2018, 10:44:58) [GCC 8.1.1 20180531]
Locale: en_US.UTF-8
Current file: '/media/C/Users/Katal/Desktop/desktop.ini'

Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/ranger/core/main.py", line 196, in main
    fm.loop()
  File "/usr/lib/python3.7/site-packages/ranger/core/fm.py", line 390, in loop
    ui.redraw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 338, in redraw
    self.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/ui.py", line 365, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/view_miller.py", line 100, in draw
    DisplayableContainer.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/displayable.py", line 256, in draw
    displayable.draw()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 176, in draw
    self._draw_file()
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/browsercolumn.py", line 203, in _draw_file
    Pager.draw(self)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 101, in draw
    self._draw_line(i, line)
  File "/usr/lib/python3.7/site-packages/ranger/gui/widgets/pager.py", line 132, in _draw_line
    self.addstr(chunk)
  File "/usr/lib/python3.7/site-packages/ranger/gui/curses_shortcuts.py", line 37, in addstr
    self.win.addstr(*args)
ValueError: embedded null character

```

**Comment by toonn** [comment]
This should've been fixed in #1091. Could you update your ranger version? The current stable release has the fix and another fix for py3.7, ranger-git has these fixes too of course.

**Comment by kosshishub** [comment]
Had this same issue on arch, 1.9.2 (from AUR) fixed this.
